doc/dev_guide/writing_docs.rst

```diff
@@ -24,6 +24,13 @@ be exposed to the end-user (usually a function or class starting with an undersc
 should still be documented to the extent that future developers can understand
 what the function does.
 
+To test the code in the docstring:
+
+.. code:: bash
```

Comment:
I moved it to the previous section (User guide documentation) and keep it separate from the doctring test.

Reply:
I just meant that I would first mention the flake8-docstrings extension because it is relevant for all docstrings. Afterwards, I would mention the examples testing, because it is relevant only for some docstrings. Just exchange these two points.